pkg/claims/result.go

```diff
@@ -66,3 +69,21 @@ func (r Result) NewOutput(name string, data []byte) Output {
 		Value:         data,
 	}
 }
+
+func (r Result) FilterSensitiveOutputs(result driver.OperationResult, bun bundle.Bundle, store secrets.Store) []Output {
```

Comment:
Since this is only used in SaveOperationResult, let's move this function closer to the code that's using it.